tests/test_emitter.py

```diff
@@ -35,7 +35,13 @@
     from watchdog.observers.inotify import InotifyFullEmitter
 elif platform.is_darwin():
     pytestmark = pytest.mark.skip('WATCHDOG-8')
```

Comment:
Because of this line, tests are skipped on macOS :see_no_evil: 
Could you have working tests when you remove it?

Reply:
> Because of this line, tests are skipped on macOS 🙈

Yeah, but Travis was all green, so the PR was good to go! 🤣

Reply:
I'm working on adding tests for all of the backends I have access to. Will hopefully have them soon.